server/app/routes/v1/auth_provider.rb

```diff
@@ -0,0 +1,65 @@
+require_relative '../../mutations/audit_logs/create'
+
+module V1
+  class AuthProviderApi < Roda
+    include RequestHelpers
+    include TokenAuthenticationHelper
+    include Auditor
+    include Logging
+
+    route do |r|
+      r.get do
+        @auth_provider = AuthProvider.instance
+        render('auth_provider/show')
+      end
+
+      r.post do
+        @auth_provider = AuthProvider.instance
+
+        validate_access_token('master_admin') unless current_user_admin?
+
+        halt_request(403, 'Access denied. You must be the local administrator.') unless current_user.email == 'admin'
```

Comment:
How about implementing `User#internal_admin?` (or similar) method instead of comparing email here?